cli/util/convert.go

```diff
@@ -16,13 +16,14 @@ import (
 
 // NewCommands returns util commands for neo-go CLI.
 func NewCommands() []cli.Command {
-	txDumpFlags := append([]cli.Flag{}, options.RPC...)
+	txDumpFlags := append([]cli.Flag{txctx.AwaitFlag}, options.RPC...)
```

Comment:
OK, we actually need transaction awaiting here because it has RPC flags and can not only dump, but also directly send transaction to the network.